bridgestyle/arcgis/togeostyler.py

```diff
@@ -411,34 +423,58 @@ def processSymbolLayer(layer, symboltype, options):
         }
 
     elif layer["type"] == "CIMVectorMarker":
-        # TODO
-        # we do not take the shape, but just the colors and stroke width
+        # Default values
+        fillColor = "#ff0000"
+        strokeColor = "#000000"
+        strokeWidth = 1.0
+        size = 10
+        wellKnownName = "circle"
+        maxX = maxY = None
+
         markerGraphics = layer.get("markerGraphics", [])
         if markerGraphics:
-            sublayers = markerGraphics[0]["symbol"]["symbolLayers"]
-            wellKnownName = to_wkt(markerGraphics[0].get("geometry"))
+            # TODO: support multiple marker graphics
+            markerGraphic = markerGraphics[0]
+            marker = processSymbolReference(markerGraphic, {})[0]
+            sublayers = [sublayer for sublayer in markerGraphic["symbol"]["symbolLayers"] if sublayer["enable"]]
             fillColor = _extractFillColor(sublayers)
             strokeColor, strokeWidth = _extractStroke(sublayers)
-        else:
-            fillColor = "#ff0000"
-            strokeColor = "#000000"
-        layer = {
+            size = marker.get("size", 10)
+            if markerGraphic["symbol"]["type"] == "CIMPointSymbol":
+                wellKnownName = marker["wellKnownName"]
+            elif markerGraphic["symbol"]["type"] in ["CIMLineSymbol", "CIMPolygonSymbol"]:
+                shape = to_wkt(markerGraphic.get("geometry"))
+                wellKnownName = shape["wellKnownName"]
+                maxX = shape.get("maxX")
+                maxY = shape.get("maxY")
+
+        marker = {
             "opacity": 1.0,
             "rotate": 0.0,
             "kind": "Mark",
             "color": fillColor,
-            "wellKnownName": wellKnownName["wellKnownName"],
-            "size": 10,
+            "wellKnownName": wellKnownName,
+            "size": size,
             "strokeColor": strokeColor,
             "strokeWidth": strokeWidth,
             "strokeOpacity": 1.0,
             "fillOpacity": 1.0,
             "Z": 0,
         }
-        if wellKnownName.get("maxX") and wellKnownName.get("maxY"):
-            layer["maxX"] = wellKnownName["maxX"]
-            layer["maxY"] = wellKnownName["maxY"]
-        return layer
+        if maxX:
+            marker["maxX"] = maxX
+        if maxY:
+            marker["maxY"] = maxY
+        markerPlacement = layer.get("markerPlacement", {}).get("placementTemplate")
+        if markerPlacement == [12, 3]:
+            marker["strokeDasharray"] = "4 0 4 7"
+            marker["size"] = 6
+            marker["perpendicularOffset"] = -3.5
+        elif markerPlacement == [15]:
```

Comment:
Can we have an enum or something that describes these specific cases ? 
[12, 3]
[15],
...?